infrastructure/docker/Dockerfile-notebook

```diff
@@ -8,7 +8,7 @@ USER $NB_UID
 
 COPY --chown=$NB_UID:$NB_UID ./client ./qs
 RUN cd ./qs && pip install .
-RUN pip install ipywidgets circuit-knitting-toolbox pyscf matplotlib
+RUN pip install ipywidgets circuit-knitting-toolbox pyscf matplotlib qiskit-nature>=0.5.2 qiskit-ibmq-provider>=0.20.2 qiskit-aer>=0.12.0
```

Comment:
@psschwei so we could say we actually resolve #491 , I'm just going to add all the extra stuff in this PR and resolve that issue, if you approve these changes